### ISSUE TITLE
Sidebar Theme

### ISSUE THREAD

**Issue by julianpop** [SachaG/SidebarFeedback#50]
Sacha, any chance you would package Sidebar as a theme and sell it? Same features. I would be interested.
Also, are the images pulled dynamically from the original post or you do some extra work on them? Nice job BTW.

**Comment by SachaG** [comment]
No, sorry. As far as the features as concerned though everything is already open-source. And yes, images are pulled dynamically.

**Comment by julianpop** [comment]
I really liked your old Sidebar theme. It's a shame to waste perfectly good theme. Rescue it from out of the clutches of the Leridans! Make it available.
